### PR TITLE
Expose version in a consumable way (Fixes #2687)

### LINKED ISSUE #2687: Expose version API in a consumable way
Currently we have `DS_PrintVersions` which is basically just a convenience function for our clients. As we're working towards API stabilization and things like a model downloader/installer, it becomes useful to be able to utilize the version information from the API.

Simplest possible solution is to just return the version string instead of printing it. Then API consumers would have to parse the result. Another possibility is to do something like what the TensorFlow Python module has, and separate the information like this (eg. for DeepSpeech v0.6.1):

VERSION = '0.6.1'
TF_VERSION = '1.14.0'
GIT_VERSION = 'v0.6.1-0-g3df20fe'
TF_GIT_VERSION = 'v1.14.0-21-ge77504ac6b'

This means the semver-style info is easily available in the _VERSION constants, and the full git version can be used by clients/diagnostics if needed.

One example use case: we write a downloader tool for models, and it can check the version of the model being downloaded against the version of the `deepspeech` package that's installed, warning you if things are not compatible.

### PR DESCRIPTION
**Pull request by reuben**

### CODE REVIEW

File: taskcluster/tc-tests-utils.sh [hunk 1]
```diff
@@ -306,24 +306,13 @@ assert_correct_warning_upsampling()
   assert_shows_something "$1" "erratic speech recognition"
 }
 
-assert_tensorflow_version()
```
Comment (lissyx) [inline taskcluster/tc-tests-utils.sh]: This was useful to ensure we don't make mistake with the TensorFlow / DeepSpeech repo dance. And we still expose the TensorFlow version in the native C++ client, so we should keep that.

Comment (reuben) [reply taskcluster/tc-tests-utils.sh]: With this PR the TF version is no longer exposed. It's compiled into libdeepspeech.so, but not exposed.

Comment (reuben) [reply taskcluster/tc-tests-utils.sh]: Which was your suggestion: https://github.com/mozilla/DeepSpeech/issues/2687#issuecomment-577629637

Comment (lissyx) [reply taskcluster/tc-tests-utils.sh]: Don't we expose here? https://github.com/mozilla/DeepSpeech/pull/2745/files#diff-5c1b1b3f0263742d08fb2b43f954fed5R265

Comment (reuben) [reply taskcluster/tc-tests-utils.sh]: Right, that's *built* into libdeepspeech.so, but not exposed anywhere. It used to be printed to stderr from `DS_PrintVersions()`. I can add a `DS_TensorFlowVersion()` API too if you feel it's useful to keep, I was just going from our discussion in issue #2687.

Comment (reuben) [reply taskcluster/tc-tests-utils.sh]: Aaah, I see what you mean, sorry. We make the test load a model and then parse stderr as before.

Comment (reuben) [reply taskcluster/tc-tests-utils.sh]: Is there a consistent model file that will be available on all tests? We vary between `${TASKCLUSTER_TMP__DIR}` and `${DATA_TMP_DIR}` for TFLite.

Comment (lissyx) [reply taskcluster/tc-tests-utils.sh]: > Is there a consistent model file that will be available on all tests? We vary between `${TASKCLUSTER_TMP__DIR}` and `${DATA_TMP_DIR}` for TFLite.

That's only going to be an issue on Android I guess, but I don't think it's a big deal, we don't need a model for `deepspeech --help`, so you should not care ?

Comment (reuben) [reply taskcluster/tc-tests-utils.sh]: I just disabled the version test on Android. `deepspeech --help` does not print the TF version.

**Pull request closed without merging**